data-tool/flows/tombstone/tombstone_mappings.py

```diff
@@ -216,8 +216,8 @@ def has_value(cls, value):
     EventFilings.CONVDSL_NULL: 'conversion',  # TODO: liquidation
     EventFilings.CONVDSO_NULL: ['conversion', ('dissolution', 'unknown')],
     EventFilings.CONVICORP_NULL: 'conversion',
-    EventFilings.CONVID1_NULL: 'conversion',  # TODO: related to invol dissolution
-    EventFilings.CONVID2_NULL: 'conversion',  # TODO: related to invol dissolution
+    EventFilings.CONVID1_NULL: ['conversion', 'putBackOn'],  # TODO: to confirm
+    EventFilings.CONVID2_NULL: ['conversion', 'putBackOn'],  # TODO: to confirm
```

Comment:
The legacy system behaviour for conversion in-dissolution status is weird. And David R has already asked home team to trace the code. I'll keep this change until being told to revert.